### DISCUSSION TITLE
Customizing Columns for Board

### DISCUSSION THREAD

**Discussion by EthanMW2000** [cloudscape-design/components#1339] [Q&A] [answered]
Noticing for this demo https://cloudscape.design/examples/react/configurable-dashboard.html the 6 columns are only available for widths larger than 2160 px. Is it possible to configure this breakpoint to allow for smaller screens to make use of the 6 columns?

Also this article https://cloudscape.design/patterns/general/service-dashboard/configurable-dashboard/ talks about the column layout for this being based on the 12 column grid system is it possible for the Board component to make use of all 12 columns or is locked to the 6 columns?

**Comment by rubencarvalho** [comment, marked as answer]
Hello Ethan!

It currently is not possible to customize the column configuration of the dashboards, nor do we have plans to introduce this feature.

Is there something about the current responsive behaviour that isn't working for your use case?

Thank you.

**Reply by EthanMW2000** [reply]
More so was wanting a little more freedom in deciding sizing and placement, but you answered my questions. Thanks for the help!

**Comment by joeprisk** [comment]
Hi @rubencarvalho 
Has this, or is this likely to change at all in the near future?

We are also looking to be able to add more columns than what is available and a bit more control on column size than what is currently available.

cheers

**Comment by mahayash315** [comment]
We’ve encountered the same limitation in our dashboard: we need finer control over widget widths and column counts at smaller container sizes.

We opened [board-components#463](https://github.com/cloudscape-design/board-components/pull/463), proposing an optional `columnLayout` prop to configure column counts at the existing responsive breakpoints, while preserving current behavior when omitted. It’s also related to [components#3256](https://github.com/cloudscape-design/components/issues/3256).

We recognize that the contribution guidelines currently accept only bug fixes. We’re sharing the implementation as a concrete proposal for discussion—would the team be open to revisiting this capability, or is there a preferred approach for this use case?
